JSON.md

```diff
@@ -16,6 +16,85 @@ All these 3 share 1 standard format, commonly called `cfbs.json`.
 Used by `cfbs add` and `cfbs search`, when index key is present in `cfbs.json` in the current working directory.
 When adding a module by URL, which has a `cfbs.json` inside of it, the index in that file should be ignored.
 
+## The process of building modules from a project into a policy set
+
+When you build a project with the `cfbs build` command, it loops through all modules in the project (`"build"` key), in order.
+Within each module it runs the individual build steps, specified in the `"steps"` key.
+
+As an example, here is a local policy file module with 1 build step:
+
+```json
+{
+  "name": "./policy.cf",
+  "description": "Local policy file added using cfbs command line",
+  "tags": ["local"],
+  "dependencies": ["autorun"],
+  "steps": ["copy ./policy.cf services/autorun/policy.cf"],
+  "added_by": "cfbs add"
+}
```

Comment:
Local policy files no longer depend on autorun. We should update this example to adhere to the modern world of cfbs.
```suggestion
{
  "name": "./policy.cf",
  "description": "Local policy file added using cfbs command line",
  "tags": ["local"],
  "added_by": "cfbs add",
  "steps": [
    "copy ./policy.cf services/cfbs/policy.cf",
    "policy_files ./policy.cf",
    "bundles my_bundle"
  ]
}
```